### PR TITLE
Add progress messages and remove new line from response

### PR DESCRIPTION
**Pull request by dukeng**
- Log some progress messages while fetching services' endpoints
- Remove newline character in return.

### CODE REVIEW

File: trace/cloud-trace-demo-app/setup.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ echo "Fetching the external IP of service a"
 endpoint=""
 for run in {1..20}
 do
+  echo "Attempt #$run to fetch the external IP of service a..."
```
Comment (tmatsuo) [inline trace/cloud-trace-demo-app/setup.sh:12]: Can you use braces like `${run}`?

File: trace/cloud-trace-demo-app/setup.sh [hunk 2]
```diff
@@ -31,6 +32,7 @@ echo "Fetching the external IP of service b"
 endpoint=""
 for run in {1..20}
 do
+  echo "Attempt #$run to fetch the external IP of service b..."
```
Comment (tmatsuo) [inline trace/cloud-trace-demo-app/setup.sh:35]: ditto

File: trace/cloud-trace-demo-app/setup.sh (hunk 1 above)
Comment (dukeng) [reply trace/cloud-trace-demo-app/setup.sh]: Done.

File: trace/cloud-trace-demo-app/setup.sh (hunk 2 above)
Comment (dukeng) [reply trace/cloud-trace-demo-app/setup.sh]: Done.

**Comment by dukeng** [comment]
@tmatsuo Do you know when the python-samples-owners will review this PR? Thanks!

**Comment by tmatsuo** [comment]
@dukeng Let's try `blunderbuss: assign` label

**Comment by tmatsuo** [comment]
I think I should be one of the `python-samples-owners`... @leahecole Do you have a power to add me there?

**Comment by dukeng** [comment]
Friendly ping @leahecole and @tmatsuo. Could I get an approval for this PR? Our team is having some documentation changes  and this PR is blocking the submission of it. Thanks!

**Comment by leahecole** [comment]
@tmatsuo you already are and are getting things auto-assigned as part of the blunderbuss rotation! @dukeng for future reference, if your assigned reviewer is a little slow, you can either use that blunderbuss label to reassign, or you can always feel free to ping them to see where they're at.

File: trace/cloud-trace-demo-app/app/app.py [hunk 3]
```diff
@@ -32,6 +33,7 @@
 propagator = google_cloud_format.GoogleCloudFormatPropagator()
```
Comment (leahecole) [inline trace/cloud-trace-demo-app/app/app.py:33]: in the docs, this line is included in the region tag. Should it be? if so, adjust the region tag to encompass it

File: trace/cloud-trace-demo-app/app/app.py [hunk 4]
```diff
@@ -72,5 +75,8 @@ def template_test():
     args = parser.parse_args()
```
Comment (leahecole) [inline trace/cloud-trace-demo-app/app/app.py:75]: Lines 61 to 64 (the `trace_context_header`) should have a region tag for the docs even if they're just illustrative

Comment (dukeng) [reply trace/cloud-trace-demo-app/app/app.py:76]: Done

File: trace/cloud-trace-demo-app/app/app.py (hunk 3 above)
Comment (dukeng) [reply trace/cloud-trace-demo-app/app/app.py:34]: Done

**Comment by dukeng** [comment]
@leahecole It seems like I need just one more approving review. Could you approve this change?

**Pull request merged**